### ISSUE TITLE
Don't apply multiple enchantment glints for items with multiple texture layers

### ISSUE THREAD

**Issue by muzikbike** [jellysquid3/Sodium#2486]
### Request Description

This is a somewhat annoying vanilla bug that notably affects enchanted leather armour. The resource pack attached to https://bugs.mojang.com/browse/MC-166101 makes this issue more clear.

The expected result would be for the enchantment glint to cover the entire item, but only once - the intensity should be constant throughout, and said intensity should only be a single layer of the glint effect.

Here's the enchanted books from that ticket:
![2024-05-09_10 29 40](https://github.com/CaffeineMC/sodium-fabric/assets/52297970/e6167e92-e48e-411a-9da6-effb89af52bd)
![2024-05-09_10 30 30](https://github.com/CaffeineMC/sodium-fabric/assets/52297970/ff1b2c13-acb5-4497-b549-4be0900705fa)
![2024-05-09_10 30 55](https://github.com/CaffeineMC/sodium-fabric/assets/52297970/e98541c6-4808-4e54-ba25-55e336d889fa)

**Comment by jellysquid3** [comment]
Closing in favor of #2497 since the issues you have reported about enchantment glint are fundamentally the same problem.
